app/src/main/java/com/kamron/pogoiv/ScreenGrabber.java

```diff
@@ -58,6 +62,35 @@ public static ScreenGrabber getInstance() {
         return instance;
     }
 
+    @NonNull private static Rect getBitmapBounds(Bitmap bmp) {
+        return new Rect(0, 0, bmp.getWidth(), bmp.getHeight());
+    }
+
+    static @Nullable @ColorInt int[] getPixelsSafe(Bitmap bmp, Point[] points) {
```

Comment:
The points are valid for portrait images, not for landscape images, and we totally got crashes from landscape images when users rotated their phones.